backends/text-to-speech/macosx/macosx-text-to-speech.mm

```diff
@@ -112,12 +113,16 @@ - (void)ignoreNextFinishedSpeaking:(BOOL)ignore {
 #endif
 	}
 
-	_messageQueue.push(SpeechText(text, encoding));
+	_messageQueue.push(SpeechText(strToSpeak, encoding));
 	if (!isSpeaking())
 		startNextSpeech();
 	return true;
 }
 
+bool MacOSXTextToSpeechManager::say(const Common::String &text, Action action, Common::String encoding) {
+	return say(Common::U32String(text), action, encoding);
```

Comment:
Why reimplement that function in `MacOSXTextToSpeechManager`? The implementation is identical to the one in the base class so I don't see the point.
(and it is identically incorrect - see my comment for the base class implementation)